### PR TITLE
Implemented nk.pl auth strategy

### PR DESCRIPTION
**Pull request by mrblur**
Implemented nk.pl nkConnect, second biggest social networking site in Poland.
It uses oauth2 for authorization and oauth 1.0 for getting user's profile data.

### CODE REVIEW

File: lib/auth.strategies/nkConnect.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                                         if( error ) callback(error);
+                                         else {
+                                           request.session["access_token"]= access_token;
+                                           if( refresh_token ) request.session["refresh_token"]= refresh_token;
+
+					my._rest_oAuth._performSecureRequest(null, '', 'GET', "http://opensocial.nk-net.pl/v09/social/rest/people/@me?fields=id,displayName,thumbnailUrl&nk_token="+request.session['access_token'], {}, '', 'application/json', function(error, data, response) {
+                                             if( error ) {
+                                               self._nkConnect_fail(callback);
+                                             }else {
+                                               console.dir(JSON.parse(data).entry);
```
Comment (mrblur) [inline lib/auth.strategies/nkConnect.js]: This shouldn't be here, sorry ;(

**Comment by ciaranj** [comment]
oh :( Do you no longer want me to pull this request in ?

**Comment by mrblur** [comment]
Sorry for that, we have decided to change naming convention and I need to first move those changes back to my fork.
I will open another Pull Request today after I will double check the code.

**Comment by ciaranj** [comment]
Its no problem, I was just going to merge it when I noted that it was closed ! :)

**Pull request closed without merging**